data/transit/route/subway.json

```diff
@@ -23,10 +23,14 @@
     {
       "displayName": "Delhi Metro",
       "id": "delhimetro-907487",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["in"]},
       "tags": {
         "network": "Delhi Metro",
-        "operator": "Delhi Metro",
+        "network:wikidata": "Q271195",
+        "network:wikipedia": "hi:दिल्ली_मेट्रो_रेल",
+        "operator": "Delhi Metro Rail Corporation Limited",
+        "operator:wikidata": "Q1029339",
+        "operator:wikipedia": "en:Delhi Metro Rail Corporation",
```

Comment:
There isn't one for this.

Reply:
Yeah, we try to use local language unless it's not available or the local page is clearly a stub.
Really the `*:wikipedia` tags are somewhat redundant once the `*:wikidata` is there, but some people and softwares will linkify them, so we keep them around.